### DISCUSSION TITLE
Server always freezing.

### DISCUSSION THREAD

**Discussion by Fwizzz** [apohl79/audiogridder#426] [Help]
I have try it with Logic 10.5.1 and Luna 1.1.6 on Mojave.

The server is a Mac mini late 2009 running El Capitan.

Plugins tested: Plugin Alliance SSL 4000 and Valhalla Plate Reverb. Buffer:8x

The server software is always freezing (turn in blue). I can setup only 3 or 4 plugins. (CPU usage: 13%) before server freeze.

There is another way to reset the server than to restart the computer?

**Comment by CaBuffalo** [comment]
Hi, what's your network connection?

The server software is always freezing (turn in blue) --- What does this
mean?

Are you using any antivirus software?

On Tue, Mar 9, 2021 at 2:48 PM Fwizzz <notifications@github.com> wrote:

> I have try it with Logic 10.5.1 and Luna 1.1.6 on Mojave.
>
> The server is a Mac mini late 2009 running El Capitan.
>
> Plugins tested: Plugin Alliance SSL 4000 and Valhalla Plate Reverb.
> Buffer:8x
>
> The server software is always freezing (turn in blue). I can setup only 3
> or 4 plugins. (CPU usage: 13%) before server freeze.
>
> There is another way to reset the server than to restart the computer?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/426>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6ACNFGXE5U35HTVUF3TC2JSPANCNFSM4Y4UP2WQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by Fwizzz** [comment]
Both computers are connected to the same HUB with a ethernet wire.

When I transfer a file via "Screen sharing", it goes up to 24 MB/sec.

"Turn in blue": the mouse cursor become a blue wheel that is spinning when I put the mouse over a plugin window or over the top right menu icon.





______________________________________________________
Michelle Lambert
Noir Silence / Les Ticky Jones / Emilie Bouchard / Studio ML
C.418 222-2866
Facebook: http://goo.gl/rbWkhP
______________________________________________________

**Comment by Fwizzz** [comment]
Wait; I investigate about a deffective Ethernet wire...




______________________________________________________
Michelle Lambert
Noir Silence / Les Ticky Jones / Emilie Bouchard / Studio ML
C.418 222-2866
Facebook: http://goo.gl/rbWkhP
______________________________________________________

**Comment by CaBuffalo** [comment]
My guess is the EI Capitan causes the problem.

Is it possible for you to do a diagnostic? Let's say, for example, install
reaper on EI Capitan as DAW and use your Mojave as server and see what
happens.

On Wed, Mar 10, 2021 at 6:23 AM Fwizzz <notifications@github.com> wrote:

> Wait; I investigate about a deffective Ethernet wire...
>
>
>
>
> ______________________________________________________
> Michelle Lambert
> Noir Silence / Les Ticky Jones / Emilie Bouchard / Studio ML
> C.418 222-2866
> Facebook: http://goo.gl/rbWkhP
> ______________________________________________________
>
> > On Mar 10, 2021, at 8:52 AM, Michelle Lambert ***@***.***> wrote:
> >
> > Both computers are connected to the same HUB with a ethernet wire.
> >
> > When I transfer a file via "Screen sharing", it goes up to 24 MB/sec.
> >
> > "Turn in blue": the mouse cursor become a blue wheel that is spinning
> when I put the mouse over a plugin window or over the top right menu icon.
> >
> >
> >
> > <michelle rond.jpg>
> >
> > ______________________________________________________
> > Michelle Lambert
> > Noir Silence / Les Ticky Jones / Emilie Bouchard / Studio ML
> > C.418 222-2866
> > Facebook: http://goo.gl/rbWkhP <http://goo.gl/rbWkhP>
> > ______________________________________________________
> >
> >> On Mar 9, 2021, at 6:07 PM, CaBuffalo ***@***.*** ***@***.***>> wrote:
> >>
> >>
> >> Hi, what's your network connection?
> >>
> >> The server software is always freezing (turn in blue) --- What does this
> >> mean?
> >>
> >> Are you using any antivirus software?
> >>
> >> On Tue, Mar 9, 2021 at 2:48 PM Fwizzz ***@***.*** ***@***.***>> wrote:
> >>
> >> > I have try it with Logic 10.5.1 and Luna 1.1.6 on Mojave.
> >> >
> >> > The server is a Mac mini late 2009 running El Capitan.
> >> >
> >> > Plugins tested: Plugin Alliance SSL 4000 and Valhalla Plate Reverb.
> >> > Buffer:8x
> >> >
> >> > The server software is always freezing (turn in blue). I can setup
> only 3
> >> > or 4 plugins. (CPU usage: 13%) before server freeze.
> >> >
> >> > There is another way to reset the server than to restart the computer?
> >> >
> >> > —
> >> > You are receiving this because you are subscribed to this thread.
> >> > Reply to this email directly, view it on GitHub
> >> > <https://github.com/apohl79/audiogridder/discussions/426 <
> https://github.com/apohl79/audiogridder/discussions/426>>, or unsubscribe
> >> > <
> https://github.com/notifications/unsubscribe-auth/ARMAR6ACNFGXE5U35HTVUF3TC2JSPANCNFSM4Y4UP2WQ
> <
> https://github.com/notifications/unsubscribe-auth/ARMAR6ACNFGXE5U35HTVUF3TC2JSPANCNFSM4Y4UP2WQ
> >>
> >> > .
> >> >
> >>
> >>
> >> --
> >> Kevin Ma
> >> Supported by Gmail.
> >> —
> >> You are receiving this because you authored the thread.
> >> Reply to this email directly, view it on GitHub <
> https://github.com/apohl79/audiogridder/discussions/426#discussioncomment-457849>,
> or unsubscribe <
> https://github.com/notifications/unsubscribe-auth/ATFECQOOPKFIWR6JWEHRU53TC2S2BANCNFSM4Y4UP2WQ
> >.
> >>
> >
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/426#discussioncomment-462770>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6GELMG2XLLWK4GPDJ3TC56FRANCNFSM4Y4UP2WQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by Fwizzz** [comment]
It seams to be the ethernet cable! It seams to work!

Thank you for your answer! If I have problems in futur, I'll try that!


This thing is AMAZING!

- Allow us to not have to "freeze" tracks!
- Allow to use old plugins that don't run on the latest Mac OS or Mac processor!!
- Allow to unload processor with a lot of bus processing!

I just sent you a donation.

Thank you all for making the world better! ;-)


Little idea: It may have a option to automatically open the first plugin GUI if there is only one plugin loaded when you open the AGridder plugin window.





______________________________________________________
Michelle Lambert
Noir Silence / Les Ticky Jones / Emilie Bouchard / Studio ML
C.418 222-2866
Facebook: http://goo.gl/rbWkhP
______________________________________________________

**Comment by CaBuffalo** [comment]
Awesome !!! Thanks for your FB and donation.

Enjoy and help is to spread the words as well. :):)

It may have a option to automatically open the first plugin GUI if there is
only one plugin loaded when you open the AGridder plugin window.   -----
This will be something that Andreas will look into it.

Cheers.

On Wed, Mar 10, 2021 at 10:00 AM Fwizzz <notifications@github.com> wrote:

> It seams to be the ethernet cable! It seams to work!
>
> Thank you for your answer! If I have problems in futur, I'll try that!
>
>
> This thing is AMAZING!
>
> - Allow us to not have to "freeze" tracks!
> - Allow to use old plugins that don't run on the latest Mac OS or Mac
> processor!!
> - Allow to unload processor with a lot of bus processing!
>
> I just sent you a donation.
>
> Thank you all for making the world better! ;-)
>
>
> Little idea: It may have a option to automatically open the first plugin
> GUI if there is only one plugin loaded when you open the AGridder plugin
> window.
>
>
>
>
>
> ______________________________________________________
> Michelle Lambert
> Noir Silence / Les Ticky Jones / Emilie Bouchard / Studio ML
> C.418 222-2866
> Facebook: http://goo.gl/rbWkhP
> ______________________________________________________
>
> > On Mar 10, 2021, at 11:10 AM, CaBuffalo ***@***.***> wrote:
> >
> >
> > My guess is the EI Capitan causes the problem.
> >
> > Is it possible for you to do a diagnostic? Let's say, for example,
> install
> > reaper on EI Capitan as DAW and use your Mojave as server and see what
> > happens.
> >
> > On Wed, Mar 10, 2021 at 6:23 AM Fwizzz ***@***.***> wrote:
> >
> > > Wait; I investigate about a deffective Ethernet wire...
> > >
> > >
> > >
> > >
> > > ______________________________________________________
> > > Michelle Lambert
> > > Noir Silence / Les Ticky Jones / Emilie Bouchard / Studio ML
> > > C.418 222-2866
> > > Facebook: http://goo.gl/rbWkhP
> > > ______________________________________________________
> > >
> > > > On Mar 10, 2021, at 8:52 AM, Michelle Lambert ***@***.***> wrote:
> > > >
> > > > Both computers are connected to the same HUB with a ethernet wire.
> > > >
> > > > When I transfer a file via "Screen sharing", it goes up to 24 MB/sec.
> > > >
> > > > "Turn in blue": the mouse cursor become a blue wheel that is spinning
> > > when I put the mouse over a plugin window or over the top right menu
> icon.
> > > >
> > > >
> > > >
> > > > <michelle rond.jpg>
> > > >
> > > > ______________________________________________________
> > > > Michelle Lambert
> > > > Noir Silence / Les Ticky Jones / Emilie Bouchard / Studio ML
> > > > C.418 222-2866
> > > > Facebook: http://goo.gl/rbWkhP <http://goo.gl/rbWkhP>
> > > > ______________________________________________________
> > > >
> > > >> On Mar 9, 2021, at 6:07 PM, CaBuffalo ***@***.*** ***@***.***>>
> wrote:
> > > >>
> > > >>
> > > >> Hi, what's your network connection?
> > > >>
> > > >> The server software is always freezing (turn in blue) --- What does
> this
> > > >> mean?
> > > >>
> > > >> Are you using any antivirus software?
> > > >>
> > > >> On Tue, Mar 9, 2021 at 2:48 PM Fwizzz ***@***.*** ***@***.***>>
> wrote:
> > > >>
> > > >> > I have try it with Logic 10.5.1 and Luna 1.1.6 on Mojave.
> > > >> >
> > > >> > The server is a Mac mini late 2009 running El Capitan.
> > > >> >
> > > >> > Plugins tested: Plugin Alliance SSL 4000 and Valhalla Plate
> Reverb.
> > > >> > Buffer:8x
> > > >> >
> > > >> > The server software is always freezing (turn in blue). I can setup
> > > only 3
> > > >> > or 4 plugins. (CPU usage: 13%) before server freeze.
> > > >> >
> > > >> > There is another way to reset the server than to restart the
> computer?
> > > >> >
> > > >> > —
> > > >> > You are receiving this because you are subscribed to this thread.
> > > >> > Reply to this email directly, view it on GitHub
> > > >> > <https://github.com/apohl79/audiogridder/discussions/426 <
> > > https://github.com/apohl79/audiogridder/discussions/426>>, or
> unsubscribe
> > > >> > <
> > >
> https://github.com/notifications/unsubscribe-auth/ARMAR6ACNFGXE5U35HTVUF3TC2JSPANCNFSM4Y4UP2WQ
> > > <
> > >
> https://github.com/notifications/unsubscribe-auth/ARMAR6ACNFGXE5U35HTVUF3TC2JSPANCNFSM4Y4UP2WQ
> > > >>
> > > >> > .
> > > >> >
> > > >>
> > > >>
> > > >> --
> > > >> Kevin Ma
> > > >> Supported by Gmail.
> > > >> —
> > > >> You are receiving this because you authored the thread.
> > > >> Reply to this email directly, view it on GitHub <
> > >
> https://github.com/apohl79/audiogridder/discussions/426#discussioncomment-457849
> >,
> > > or unsubscribe <
> > >
> https://github.com/notifications/unsubscribe-auth/ATFECQOOPKFIWR6JWEHRU53TC2S2BANCNFSM4Y4UP2WQ
> > > >.
> > > >>
> > > >
> > >
> > > —
> > > You are receiving this because you commented.
> > > Reply to this email directly, view it on GitHub
> > > <
> https://github.com/apohl79/audiogridder/discussions/426#discussioncomment-462770
> >,
> > > or unsubscribe
> > > <
> https://github.com/notifications/unsubscribe-auth/ARMAR6GELMG2XLLWK4GPDJ3TC56FRANCNFSM4Y4UP2WQ
> >
> > > .
> > >
> >
> >
> > --
> > Kevin Ma
> > Supported by Gmail.
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly, view it on GitHub <
> https://github.com/apohl79/audiogridder/discussions/426#discussioncomment-463640>,
> or unsubscribe <
> https://github.com/notifications/unsubscribe-auth/ATFECQINBVBMMKGRJLD3PU3TC6KWHANCNFSM4Y4UP2WQ
> >.
> >
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/426#discussioncomment-464382>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6BH6FVAVO2ZM5YS3VLTC6XTVANCNFSM4Y4UP2WQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by Fwizzz** [comment]
Pleasure!

It is below the price of a 420mhz SHARC chip (Sold by UA)! and it go 8x faster!




______________________________________________________
Michelle Lambert
Noir Silence / Les Ticky Jones / Emilie Bouchard / Studio ML
C.418 222-2866
Facebook: http://goo.gl/rbWkhP
______________________________________________________

**Comment by CaBuffalo** [comment]
More features and developments are under the way. Please stay tuned.

Cheers.

On Wed, Mar 10, 2021 at 2:45 PM Fwizzz <notifications@github.com> wrote:

> Pleasure!
>
> It is below the price of a 420mhz SHARC chip (Sold by UA)! and it go 8x
> faster!
>
>
>
>
> ______________________________________________________
> Michelle Lambert
> Noir Silence / Les Ticky Jones / Emilie Bouchard / Studio ML
> C.418 222-2866
> Facebook: http://goo.gl/rbWkhP
> ______________________________________________________
>
> > On Mar 10, 2021, at 5:39 PM, CaBuffalo ***@***.***> wrote:
> >
> >
> > Awesome !!! Thanks for your FB and donation.
> >
> > Enjoy and help is to spread the words as well. :):)
> >
> > It may have a option to automatically open the first plugin GUI if there
> is
> > only one plugin loaded when you open the AGridder plugin window. -----
> > This will be something that Andreas will look into it.
> >
> > Cheers.
> >
> > On Wed, Mar 10, 2021 at 10:00 AM Fwizzz ***@***.***> wrote:
> >
> > > It seams to be the ethernet cable! It seams to work!
> > >
> > > Thank you for your answer! If I have problems in futur, I'll try that!
> > >
> > >
> > > This thing is AMAZING!
> > >
> > > - Allow us to not have to "freeze" tracks!
> > > - Allow to use old plugins that don't run on the latest Mac OS or Mac
> > > processor!!
> > > - Allow to unload processor with a lot of bus processing!
> > >
> > > I just sent you a donation.
> > >
> > > Thank you all for making the world better! ;-)
> > >
> > >
> > > Little idea: It may have a option to automatically open the first
> plugin
> > > GUI if there is only one plugin loaded when you open the AGridder
> plugin
> > > window.
> > >
> > >
> > >
> > >
> > >
> > > ______________________________________________________
> > > Michelle Lambert
> > > Noir Silence / Les Ticky Jones / Emilie Bouchard / Studio ML
> > > C.418 222-2866
> > > Facebook: http://goo.gl/rbWkhP
> > > ______________________________________________________
> > >
> > > > On Mar 10, 2021, at 11:10 AM, CaBuffalo ***@***.***> wrote:
> > > >
> > > >
> > > > My guess is the EI Capitan causes the problem.
> > > >
> > > > Is it possible for you to do a diagnostic? Let's say, for example,
> > > install
> > > > reaper on EI Capitan as DAW and use your Mojave as server and see
> what
> > > > happens.
> > > >
> > > > On Wed, Mar 10, 2021 at 6:23 AM Fwizzz ***@***.***> wrote:
> > > >
> > > > > Wait; I investigate about a deffective Ethernet wire...
> > > > >
> > > > >
> > > > >
> > > > >
> > > > > ______________________________________________________
> > > > > Michelle Lambert
> > > > > Noir Silence / Les Ticky Jones / Emilie Bouchard / Studio ML
> > > > > C.418 222-2866
> > > > > Facebook: http://goo.gl/rbWkhP
> > > > > ______________________________________________________
> > > > >
> > > > > > On Mar 10, 2021, at 8:52 AM, Michelle Lambert ***@***.***> wrote:
> > > > > >
> > > > > > Both computers are connected to the same HUB with a ethernet
> wire.
> > > > > >
> > > > > > When I transfer a file via "Screen sharing", it goes up to 24
> MB/sec.
> > > > > >
> > > > > > "Turn in blue": the mouse cursor become a blue wheel that is
> spinning
> > > > > when I put the mouse over a plugin window or over the top right
> menu
> > > icon.
> > > > > >
> > > > > >
> > > > > >
> > > > > > <michelle rond.jpg>
> > > > > >
> > > > > > ______________________________________________________
> > > > > > Michelle Lambert
> > > > > > Noir Silence / Les Ticky Jones / Emilie Bouchard / Studio ML
> > > > > > C.418 222-2866
> > > > > > Facebook: http://goo.gl/rbWkhP <http://goo.gl/rbWkhP>
> > > > > > ______________________________________________________
> > > > > >
> > > > > >> On Mar 9, 2021, at 6:07 PM, CaBuffalo ***@***.*** ***@***.***>>
> > > wrote:
> > > > > >>
> > > > > >>
> > > > > >> Hi, what's your network connection?
> > > > > >>
> > > > > >> The server software is always freezing (turn in blue) --- What
> does
> > > this
> > > > > >> mean?
> > > > > >>
> > > > > >> Are you using any antivirus software?
> > > > > >>
> > > > > >> On Tue, Mar 9, 2021 at 2:48 PM Fwizzz ***@***.*** ***@***.***>>
> > > wrote:
> > > > > >>
> > > > > >> > I have try it with Logic 10.5.1 and Luna 1.1.6 on Mojave.
> > > > > >> >
> > > > > >> > The server is a Mac mini late 2009 running El Capitan.
> > > > > >> >
> > > > > >> > Plugins tested: Plugin Alliance SSL 4000 and Valhalla Plate
> > > Reverb.
> > > > > >> > Buffer:8x
> > > > > >> >
> > > > > >> > The server software is always freezing (turn in blue). I can
> setup
> > > > > only 3
> > > > > >> > or 4 plugins. (CPU usage: 13%) before server freeze.
> > > > > >> >
> > > > > >> > There is another way to reset the server than to restart the
> > > computer?
> > > > > >> >
> > > > > >> > —
> > > > > >> > You are receiving this because you are subscribed to this
> thread.
> > > > > >> > Reply to this email directly, view it on GitHub
> > > > > >> > <https://github.com/apohl79/audiogridder/discussions/426 <
> > > > > https://github.com/apohl79/audiogridder/discussions/426>>, or
> > > unsubscribe
> > > > > >> > <
> > > > >
> > >
> https://github.com/notifications/unsubscribe-auth/ARMAR6ACNFGXE5U35HTVUF3TC2JSPANCNFSM4Y4UP2WQ
> > > > > <
> > > > >
> > >
> https://github.com/notifications/unsubscribe-auth/ARMAR6ACNFGXE5U35HTVUF3TC2JSPANCNFSM4Y4UP2WQ
> > > > > >>
> > > > > >> > .
> > > > > >> >
> > > > > >>
> > > > > >>
> > > > > >> --
> > > > > >> Kevin Ma
> > > > > >> Supported by Gmail.
> > > > > >> —
> > > > > >> You are receiving this because you authored the thread.
> > > > > >> Reply to this email directly, view it on GitHub <
> > > > >
> > >
> https://github.com/apohl79/audiogridder/discussions/426#discussioncomment-457849
> > > >,
> > > > > or unsubscribe <
> > > > >
> > >
> https://github.com/notifications/unsubscribe-auth/ATFECQOOPKFIWR6JWEHRU53TC2S2BANCNFSM4Y4UP2WQ
> > > > > >.
> > > > > >>
> > > > > >
> > > > >
> > > > > —
> > > > > You are receiving this because you commented.
> > > > > Reply to this email directly, view it on GitHub
> > > > > <
> > >
> https://github.com/apohl79/audiogridder/discussions/426#discussioncomment-462770
> > > >,
> > > > > or unsubscribe
> > > > > <
> > >
> https://github.com/notifications/unsubscribe-auth/ARMAR6GELMG2XLLWK4GPDJ3TC56FRANCNFSM4Y4UP2WQ
> > > >
> > > > > .
> > > > >
> > > >
> > > >
> > > > --
> > > > Kevin Ma
> > > > Supported by Gmail.
> > > > —
> > > > You are receiving this because you authored the thread.
> > > > Reply to this email directly, view it on GitHub <
> > >
> https://github.com/apohl79/audiogridder/discussions/426#discussioncomment-463640
> >,
> > > or unsubscribe <
> > >
> https://github.com/notifications/unsubscribe-auth/ATFECQINBVBMMKGRJLD3PU3TC6KWHANCNFSM4Y4UP2WQ
> > > >.
> > > >
> > >
> > > —
> > > You are receiving this because you commented.
> > > Reply to this email directly, view it on GitHub
> > > <
> https://github.com/apohl79/audiogridder/discussions/426#discussioncomment-464382
> >,
> > > or unsubscribe
> > > <
> https://github.com/notifications/unsubscribe-auth/ARMAR6BH6FVAVO2ZM5YS3VLTC6XTVANCNFSM4Y4UP2WQ
> >
> > > .
> > >
> >
> >
> > --
> > Kevin Ma
> > Supported by Gmail.
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly, view it on GitHub <
> https://github.com/apohl79/audiogridder/discussions/426#discussioncomment-465784>,
> or unsubscribe <
> https://github.com/notifications/unsubscribe-auth/ATFECQJQSNIDS7LEINIB74DTC7YLBANCNFSM4Y4UP2WQ
> >.
> >
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/426#discussioncomment-465804>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6GWP6DJZKN3QMOUKVDTC7Y7JANCNFSM4Y4UP2WQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.
